samples/04.0-file-transfer/consumer/build.gradle.kts

```diff
@@ -27,7 +27,6 @@ dependencies {
 
     implementation(project(":extensions:in-memory:assetindex-memory"))
     implementation(project(":extensions:in-memory:transfer-store-memory"))
-    implementation(project(":extensions:in-memory:assetindex-memory"))
     implementation(project(":extensions:in-memory:negotiation-store-memory"))
```

Comment:
Take a look two lines above ;-) It was something I identified while trying the sample in the onboarding progress and thought a own PR for this would be an overkill

Reply:
Whups, didn't see it in the diff. Fell free to resolve then.